simulator/modules/postgres.py

```diff
@@ -209,7 +209,7 @@ def _single_insert_mode(events, use_multiple_tables, batch_size, batch_mode):
 
 
 _indices = [
-    "CREATE INDEX IF NOT EXISTS events_device_ts ON events (device_id, timestamp ASC)",
+    "CREATE INDEX IF NOT EXISTS events_device_ts ON events (device_id ASC, timestamp ASC, temperature ASC)",
```

Comment:
temperature should better be outside of "sorted" columns like in

```sql
CREATE INDEX IF NOT EXISTS events_device_ts ON events (device_id ASC, timestamp ASC) INCLUDE(temperature)
```

Reply:
If the value would change, then it would be less work for the index maintenance

I agree the diff is small, however, moving the column to `INCLUDE` would better hint the intention to the readers